python/tests/test_topology.py

```diff
@@ -49,6 +49,532 @@ def generate_segments(n, sequence_length=100, seed=None):
     return segs
 
 
+def kc_distance_tree(tree1, tree2, topo_v_age=0):
+    """
+    Returns the Kendall-Colijn distance between the specified
+    pair of trees. topology_vs_age determines weight of topology vs
```

Comment:
Discrepancy between `topo_v_age` and `topology_v_age`

Reply:
Good catch